testing/events.go

```diff
@@ -120,6 +120,22 @@ func ParsePacketFromEvents(events []abci.Event) (channeltypes.Packet, error) {
 	return channeltypes.Packet{}, fmt.Errorf("acknowledgement event attribute not found")
 }
 
+// ParsePacketsFromEvents parses events emitted from a MsgRecvPacket and returns
+// all the packets found.
+func ParsePacketsFromEvents(events []abci.Event) ([]channeltypes.Packet, error) {
```

Comment:
I'd actually prefer if the main logic was happening in this function and have `ParsePacketFromEvents` would call into this and pick the first element from the result. I'm ok either way though.

Reply:
Yeah, the implementation might become easier to reason like that.

Reply:
Good idea, so I added a test and then moved the logic from `ParsePacketFromEvents` to `ParsePacketsFromEvents`.